packages/cards-editor-config-writer/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "@sap-ux/cards-editor-config-writer",
+    "version": "0.2.1",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/cards-editor-config-writer"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Acards-editor-config-writer"
+    },
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "tsc --build",
+        "watch": "tsc --watch",
+        "clean": "rimraf --glob dist test/test-output *.tsbuildinfo",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors"
+    },
+    "files": [
+        "LICENSE",
+        "dist"
+    ],
+    "dependencies": {
+        "@sap-ux/ui5-config": "workspace:*",
+        "@sap-ux/project-access": "workspace:*",
+        "mem-fs": "2.1.0",
+        "mem-fs-editor": "9.4.0"
+    },
+    "devDependencies": {
+        "@types/mem-fs": "1.1.2",
+        "@types/mem-fs-editor": "7.0.1"
+    },
+    "engines": {
+        "pnpm": ">=6.26.1 < 7.0.0 || >=7.1.0",
```

Comment:
```suggestion
```

We removed the mention of `pnpm` in `engines` section of `package.json` on module level and keep it only on root for compatibility reasons, see also PR https://github.com/SAP/open-ux-tools/pull/1688